apps/studio/src/features/editing-experience/components/Preview.tsx

```diff
@@ -48,11 +49,6 @@ function SuspendablePreview({
         siteMap: { title: "Home", permalink: "/", children: [] },
         environment: "production",
         // TODO: Fetch from DB in the future
```

Comment:
nit: this comment should move up together with the code change?

Reply:
sounds good